src/software/amazon/ion/impl/lite/IonValueLite.java

```diff
@@ -661,14 +661,16 @@ public void setTypeAnnotations(String... annotations)
     public final boolean hasTypeAnnotation(String annotation)
     {
         if (annotation != null && annotation.length() > 0) {
-            int pos = find_type_annotation(annotation);
+            int pos = findTypeAnnotation(annotation);
             if (pos >= 0) {
```

Comment:
I realize this is not part of your CR, but there are quite a few
```java
if (condition) {
  return true;
} else {
  return false;
}
```
-ish statements that could be reduced to `return condition;`.